lib/Zonemaster/Backend/Config.pm

```diff
@@ -186,6 +186,22 @@ sub lock_on_queue {
     return $val;
 }
 
+sub timeout_before_retrying_test_in_minutes {
+    my ($self) = @_;
+
+    my $val = $self->{cfg}->val( 'ZONEMASTER', 'timeout_before_retrying_test_in_minutes' );
+
+    return ($val)?($val):(10);
+}
+
+sub maximal_number_of_retries {
+    my ($self) = @_;
+
+    my $val = $self->{cfg}->val( 'ZONEMASTER', 'maximal_number_of_retries' );
```

Comment:
I think this could be shorter too, e.g. "max_retries".

This method needs documentation in POD below.

Reply:
I really prefer very long self explanatory variables that do not require to read documentation do be understood.

Adding POD for the method.

Reply:
Yeah, but the variable name does not need to be longer needed. That increases the risk of spelling errors.

Reply:
I agree, but we live in a world of cut and paste, don't we?
There are ready made place holders in the backend_config.ini file now.